content/peloton/drone-google-chat/index.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2016-01-13T00:00:00+00:00
+title: Google chat
+author: josmo
+tags: [  notifications, chat  ]
+repo: josmo/drone-google-chat
+logo: gchat.png
```

Comment:
it is actually just the Google logo. When we don't have the product logo in SVG we just fallback to the company logo.